gateway/src/main/kotlin/com/example/gateway/web/mapper/proto/pass/CancelPassResponseMapper.kt

```diff
@@ -0,0 +1,23 @@
+package com.example.gateway.web.mapper.proto.pass
+
+import com.example.internal.input.reqreply.CancelPassResponse
+import com.example.passmanagersvc.exception.PassOwnerNotFoundException
+
+object CancelPassResponseMapper {
+    fun CancelPassResponse.toUnitResponse() {
+        require(this != CancelPassResponse.getDefaultInstance()) {
```

Comment:
I don't agree with IllegalArgumentException thrown here. It will be mapped to some 4xx error code, but in fact it's server error and it should be mapped to 5xxx